articles/service-fabric/service-fabric-reliable-services-communication-remoting.md

```diff
@@ -66,6 +66,16 @@ class MyService : StatelessService, IMyService
     }
 }
 ```
+
+To use remoting in a stateful service, we would rely on the extension method provided, and add the following call within the generated `CreateServiceReplicaListeners` method:
```

Comment:
These extension method are not needed anymore. We are encouraging users to use CreateServiceRemotingInstanceListeners extension method instead .

Here is example of this new extension method.
https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-communication-remoting#how-to-use-the-remoting-v2-stack

Plan is to use same in this sample also.
https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-communication-remoting#set-up-remoting-on-a-service.

This change will make users to use only 1 extension method for both remoting v1 and v2 , so much simpler.

Let me know if you are interested in doing this change.

Reply:
Interesting. I'm happy to have a look at that, yes.

Reply:
@suchiagicha  Hm, I've looked at my code again, and the sample, and I am calling `CreateServiceRemotingReplicaListeners()`. I'm only mentioning the `CreateServiceReplicaListeners` as that's what gets generated. That said, I see my code example has a _bug_ where I call it `CreateServiceInstanceListener` instead of `CreateServiceReplicaListeners` which is what is auto-generated by the template.